.github/workflows/publish-prod.yaml

```diff
@@ -5,6 +5,7 @@ on:
     tags:
       - "[0-9]+.[0-9]+.[0-9]+"
       - "!0.25.[0-9]+"
+      - "!0.26.[0-9]+"
```

Comment:
I'm sure I can combine these two excluded versions with a more complex glob, but I kind of like having them on separate lines.

Reply:
Visually I like the separation too.